apps/common/lib/mix/tasks/namespace.ex

```diff
@@ -36,7 +36,7 @@ defmodule Mix.Tasks.Namespace do
       IO.write("Applying namespace: #{percent_complete} complete")
     end)
 
-    Mix.Shell.IO.info("\nNamspace applied. Generating apps")
+    Mix.Shell.IO.info("\nNamespace applied. Generating apps")
```

Comment:
what, you didn't like my spelling?
At least you know _why_ I mistyped it.

Reply:
the datahand speaks